### DISCUSSION TITLE
接口问题

### DISCUSSION THREAD

**Discussion by kistonw** [jeessy2/ddns-go#929] [Q&A]
2023/12/05 22:02:58 配置文件已保存在: /root/.ddns_go_config.yaml
2023/12/05 22:02:58 请求接口 https://alidns.aliyuncs.com/ 失败! 返回内容: {"RequestId":"08F2429C-D3EA-5223-9D3E-94A0A9657263","HostId":"alidns.aliyuncs.com","Code":"IncorrectDomainUser","Message":"The domain name belongs to other users. Transfer the domain name to the current user and then try the binding and setting actions.","Recommend":"https://api.aliyun.com/troubleshoot?q=IncorrectDomainUser&product=Alidns&requestId=08F2429C-D3EA-5223-9D3E-94A0A9657263"} ,返回状态码: 400

**Comment by jeessy2** [comment]
看提示。返回的是这个域名不是你的，检查填写准确没有

**Reply by kistonw** [reply]
不是这个原因 我们只提供key 我阿里云控制台都显示解析正常

**Reply by jeessy2** [reply]
那你自己看了。

The domain name belongs to other users. Transfer the domain name to the current user and then try the binding and setting actions.

**Reply by kistonw** [reply]
查询文档说IncorrectDomainUser, 该域名非本账户域名，阿里云域名不能进行跨账号解析
可是我就这一个阿里云账号
